config/locales/en.yml

```diff
@@ -274,7 +274,7 @@ en:
         outright: Yes, owned outright
         with_mortgage: Yes, with a mortgage or loan
         none: "No"
-      guidance_on_properties: Guidance on properties (link needed)
+      guidance_on_properties: Guidance on properties (opens in new tab)
```

Comment:
Looking at the GDS guidance Lel linked to, the rule is that "(opens in new tab)" should appear _any_ time a link opens in a new tab. Since that's _every_ sidebar link, I think we should implement this by appending this as text to every link text passed to the sidebar partial, otherwise we'll have to manually go through and add this to _every_ guidance link.